libs/features/pages/about/src/lib/features-pages-about.module.ts

```diff
@@ -8,10 +8,14 @@ import { CarouselModule } from '@elewa-website/elements/layout/carousel';
 import { MainPageModule } from '@elewa-website/elements/layout/pages/main-page';
 
 import { AboutPageComponent } from './main/about-page/about-page.component';
-import { TeamMembersSectionComponent } from './components/team-members-section/team-members-section.component';
 import { DiscoverSectionComponent } from './components/discover-section/discover-section.component';
 
 import { AboutRoutingModule } from './about.routing';
+import { TeamMembersSectionComponent } from './components/team-members-section/team-members-section.component';
+import { AboutHeroSectionComponent } from './components/about-hero-section/about-hero-section.component';
+import { ImagesModule } from '@elewa-website/elements/layout/images';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
```

Comment:
Kindly, observe the order of imports; elements imports should appear before local imports